src/linear_algebra/projective_space/basic.lean

```diff
@@ -58,13 +62,27 @@ def mk (v : V) (hv : v ≠ 0) : ℙ K V := quotient.mk' ⟨v,hv⟩
 /-- A variant of `projectivization.mk` in terms of a subtype. `mk` is preferred. -/
 def mk' (v : { v : V // v ≠ 0 }) : ℙ K V := quotient.mk' v
 
-@[simp] lemma mk'_eq_mk (v : { v : V // v ≠ 0}) :
+@[simp]
+lemma mk'_eq_mk (v : { v : V // v ≠ 0}) :
   mk' K v = mk K v v.2 :=
 by { dsimp [mk, mk'], congr' 1, simp }
 
+/- Since `mk` is the preferred representation, we can `simp` others to it. -/
+
```

Comment:
I agree with this statement but we tend not to have floating comments like this.
```suggestion
```